app/build.gradle.kts

```diff
@@ -68,15 +94,25 @@ android {
         sourceCompatibility = sharedJavaVersion
         targetCompatibility = sharedJavaVersion
     }
+
+    signingConfigs {
+        getByName("debug") {
+            storeFile = file("./debug.keystore")
+        }
+    }
```

Comment:
I think you forgot to use it in the buildType. You need to reference it above.

_BTW, we might want to put the keystore file at the root of the project, and use `rootProject.file(…)` in case we want to add another application module in this project in the future._